flambe/field/text.py

```diff
@@ -274,6 +374,10 @@ def with_embeddings(
         embeddings_binary : bool, optional
             Whether the input embeddings are provided in binary format,
             by default False
+        setup_all_embeddings: bool
```

Comment:
Will this also be removed from the constructor?

Reply:
Yes, as this param only works when there are embeddings